xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Videos/WebHome.xml

```diff
@@ -136,8 +136,11 @@
 #macro (helpVideoCard $data)
   &lt;div class="videoCard"&gt;
     &lt;iframe src="$data.url" allowfullscreen title="$escapetool.xml($data.title)" 
-        aria-description="$escapetool.xml($services.localization.render('help.videos.hint'))" &gt;
+        aria-describedby="video_hint_$foreach.index" &gt;
     &lt;/iframe&gt;
+    &lt;span id="video_hint_$foreach.index" class="sr-only"&gt;
```

Comment:
Didn't we conclude that texts used by `aria-describedby` should use `hidden` or do you think this should be available independent of the iframe?